src/diric.jl

```diff
@@ -0,0 +1,50 @@
+# diric.jl Dirichlet kernel
+# 2021-03-19, Jeff Fessler, University of Michigan
```

Comment:
These comments are normally not included in the code. Please follow the format from the rest of the repository.

Your contribution will be visible in the git logs. And also in the contributors list generated on the GitHub website.

Reply:
done